### DISCUSSION TITLE
Jiosaavn returns empty data when requested from a server.

### DISCUSSION THREAD

**Discussion by aravind-tronix** [cyberboysumanjay/JioSaavnAPI#44] [Q&A]
Hi.. I have been trying to make a songbot in telegram using this API and successfully made one. However, when i deployed it  in server(tried in USA, Germany server locations) sadly Jiosaavn return empty data for most of the queries. I even used proxies. For example : [Tried this query](https://www.jiosaavn.com/api.php?__call=autocomplete.get&_format=json&_marker=0&cc=in&includeMetaTags=1&query=whatever%20it%20takes) it works fine when requested from server located in India but return empty data when requested from Germany based server and i cannot afford servers locations in India for now.
Is there a work around?

**Comment by aravind-tronix** [comment]
Sorry. i used the proxy wrongly. Now its working fine. Admin please feel free to delete this discussion.

**Reply by cyberboysumanjay** [reply]
Great 😄
